roboflow/core/workspace.py

```diff
@@ -61,6 +62,76 @@ def project(self, project_name):
 
         return Project(self.__api_key, dataset_info, self.model_format)
 
+    def active_learning(self, raw_data_location, raw_data_extension, inference_endpoint, upload_destination, conditionals):
+        '''
+        @params:
```

Comment:
These params don't match the signature; what's _location vs _extension?

Reply:
I had added these, just committed the new definitions